src/getfield.f

```diff
@@ -1,121 +1,99 @@
+!>    @file
+!>    @brief This subroutine obtain grid information and the unpacked data
```

Comment:
The brief statement applied to the whole file, not just the first subroutine. (In most of these files, there is only one subroutine, but in this file there is more than one.)

So it should be something like:

!> @brief Contains subroutines to get fields from...

Reply:
You are right.

Reply:
The first subroutine is the main one. Following 4 are to unpack data from section 3-6.

Reply:
OK, put a summary of the file in the file @brief...

Reply:
Okay